projects/ion/src/lib/chip/chip.component.scss

```diff
@@ -63,7 +63,8 @@
   box-sizing: border-box;
   -moz-box-sizing: border-box;
   -webkit-box-sizing: border-box;
-  border: 1.5px solid $primary-5;
+  border: 0.1px solid $primary-5;
```

Comment:
@edsonsnunes e @iurynogueira  eu não sei se isso pode ser considerado com uma boa prática. O ponto é que ao definir uma borda zerada, por conta da cor que deve ser aplicada atravez da propriedade da borda, ela meio que tem um tamanho por default.  A remoção da borda implica em uma borda preta de uma outra propriedade/estado.

Reply:
Can you explain better the actual problem?

Reply:
The border property are changing the size of the chip when selected, so by removing the border property to put only outline property on selected state, implies on a black border. Because of that, the two properties must be applied at the same time. But by default, the border property has a number, so i have to put a 0.1px border.

Reply:
I don't know if I understood the problem, but in response to "by default the border property has a number", you can change border property to `border: none;`, which will set all default values to 0 or "undefined".
If the border is changing the size of the component, you can set `box-sizing: border-box;` (as I can see, is already setted) and set a fixed height to the component, which will make the border grow to inside (from default, the box grow to outside, that's why the border makes the component bigger).

Reply:
![image](https://user-images.githubusercontent.com/8728782/199036209-5b9c4b88-f7c3-4e93-b35a-76b3a65e6309.png)
Border none makes a border from another class appears.

Reply:
And by putting a height property in the class of the state of the button, still doesn't work, and the behavior of the another buttons, with diferents sizes, bugs. 
[Gravação de tela de 31-10-2022 11:58:28.webm](https://user-images.githubusercontent.com/8728782/199039509-0df65e78-9c6e-40d4-9fc5-f9b7404ebd87.webm)